pallets/rmrk-core/src/functions.rs

```diff
@@ -184,7 +190,47 @@ impl<T: Config> Nft<T::AccountId, StringLimitOf<T>> for Pallet<T> {
 	}
 }
 
-impl<T: Config> Pallet<T> {
+impl<T: Config> Pallet<T>
+where
+	T: pallet_uniques::Config<ClassId = CollectionId, InstanceId = NftId>,
+{
+
+	pub fn nft_to_account_id<AccountId: Codec>(collection_id: CollectionId, nft_id: NftId) -> AccountId {
+		(b"RmrkNft/", collection_id, nft_id)
```

Comment:
Let's make `b"RmrkNft/"` a constant

Reply:
A Substrate constant like `MaxRecursions` or just add a constant to this file or constants file w/ a line `pub const RANDOMNESS_RMRK_NFT: &[u8] = b"RmrkNft/";`?

Reply:
Easier to make it a pub const. Can move to a configurable constant later when necessary. Just to keep it simple